apps/juicebox_web/assets/js/videos/video.scss

```diff
@@ -1,13 +1,41 @@
-.video-wrapper {
+.video-wrapper,
+.video-container {
   position: relative;
-  padding-bottom: 56.25%; /* 16:9 */
-  padding-top: 25px;
-  height: 0;
 }
+
+.video-expanded.video-present {
+  .video-wrapper {
+    position: relative;
+    padding-bottom: 56.25%; /* 16:9 */
```

Comment:
Properties should be ordered position, height, padding-top, padding-bottom

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)